scripts/linters/general_purpose_linter.py

```diff
@@ -581,6 +581,14 @@
         'message': 'Please use keys() instead.',
         'excluded_files': (),
         'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'\.put_async\('),
+        'message': (
+            'Please use put_async_for_human() or put_async_for_bot() '
+            'instead.'),
```

Comment:
I mean this only applies to the case when the model is a human maintained one, in other cases we want to forbid these entirnely right?

Reply:
So should I ask the developers to use put method instead?

Reply:
I would say so.

Reply:
Done.